x-pack/filebeat/docs/inputs/input-azure-blob-storage.asciidoc

```diff
@@ -385,6 +386,8 @@ filebeat.inputs:
     timestamp_epoch: 1627233600
 ----
 
+The Azure Blob Storage APIs don't provide a direct way to filter files based on the timestamp, so the input will download all the files and then filter them based on the timestamp. This can cause a bottleneck in processing if the number of files are very high. It is recommended to use this attribute only when the number of files are limited or ample resources are available.
```

Comment:
```suggestion
The Azure Blob Storage APIs don't provide a direct way to filter files based on timestamp, so the input will download all the files and then filter them based on the timestamp. This can cause a bottleneck in processing if the number of files are very high. It is recommended to use this attribute only when the number of files are limited or ample resources are available.
```